### PR TITLE
Fix Bug 1217353, adds new infobar app to bedrock

### PR DESCRIPTION
**Pull request by schalkneethling**
@pmclanahan r?

### CODE REVIEW

File: bedrock/infobar/templates/infobar/infobar.jsonp [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "message": "{{ _('Would you like to see this page in your language?')|js_escape }}",
+    {# L10n: Your translation can just say Yes -#}
+    "accept": "{{ _('Yes, please!')|js_escape }}",
+    {# L10n: Your translation can just say No -#}
+    "cancel": "{{ _('No, thanks.')|js_escape }}",
+    {# L10n: Update bar messaging -#}
+    "update-message": "{{ _('Looks like you’re using an older version of Firefox.')|js_escape }}",
+    "update-accept": "{{ _('Update to stay fast and safe.')|js_escape }}",
+    "update-cancel": "{{ _('Later')|js_escape }}",
+    "latestfx": "{{ latest_firefox_version }}",
```
Comment (pmclanahan) [inline bedrock/infobar/templates/infobar/infobar.jsonp]: Trailing comma. Bad for JS right?

File: bedrock/infobar/urls.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from django.conf.urls import url
+
+from . import views
+
```
Comment (pmclanahan) [inline bedrock/infobar/urls.py]: NIT: 2 spaces after imports.

**Comment by pmclanahan** [comment]
Looking good. Is there a separate bug/issue for adding the bar back to bedrock?

File: bedrock/infobar/templates/infobar/infobar.jsonp (hunk 1 above)
Comment (schalkneethling) [inline bedrock/infobar/templates/infobar/infobar.jsonp]: It used to have a trailing comma but, let me change that and see if all is still good.

**Comment by schalkneethling** [comment]
@pmclanahan Not yet, no.

File: bedrock/infobar/views.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+    return inner_last_modified
+
+
+def _resp(request, path, ctype, context=None):
+    resp = l10n_utils.render(request, path, context, content_type=ctype)
+
+    is_enabled = not settings.TEMPLATE_DEBUG and settings.CDN_BASE_URL
+    if is_enabled and isinstance(resp, HttpResponseRedirect):
+        # CDN URL should be protocol relative, but that won't work
```
Comment (pmclanahan) [inline bedrock/infobar/views.py]: I don't think this CDN stuff is going to work. The CDN only works with certain URLs, and I don't remember "/infobar" being one of them. I believe "/tabzilla" was added specially. I think we're fine just letting this be a normal bedrock URL.

File: bedrock/infobar/templates/infobar/infobar.jsonp (hunk 1 above)
Comment (pmclanahan) [inline bedrock/infobar/templates/infobar/infobar.jsonp]: May be an old rule, but I think to be as broadly supportive of browsers as possible a trailing comma is still bad. I could be wrong though.

File: bedrock/infobar/views.py (hunk 3 above)
Comment (schalkneethling) [inline bedrock/infobar/views.py]: I copied this from the current code in tabzilla. Should I just remove the above function and have:

```
resp = l10n_utils.render(request, 'infobar/infobar.jsonp', 'application/javascript', content_type=ctype)
resp['Access-Control-Allow-Origin'] = '*'
return resp
```
inside

```
def infobar_jsonp(request):
```

Comment (pmclanahan) [inline bedrock/infobar/views.py]: Basically yes. It's `l10n_utils.render(request, 'infobar/infobar.jsonp', content_type='application/javascript')`.

**Comment by schalkneethling** [comment]
@pmclanahan we good to merge?

**Pull request merged**